app/main.ipynb

```diff
@@ -126,19 +170,31 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 10,
-   "outputs": [],
    "source": [
-    "array = np.array([2.3, 7.8, 3.2, 1.1, 5.8, 9.5, 17.6, 11.1])\n",
-    "# write your code here"
+    "array = np.array([2.3, 7.8, 3.2, 1.1, 5.8, 9.5, 17.6, 11.1], dtype=np.float16)\n",
+    "result = array[1] + array[5] + array[4] + array[2]\n",
+    "np.ceil(result)"
```

Comment:
The indices used in the summation do not yield the expected result of 27. Please review the indices to ensure they sum up to 27 as specified in the task description.